parameter/CMakeLists.txt

```diff
@@ -103,8 +105,13 @@ add_library(parameter SHARED
     XmlFileIncluderElement.cpp
     XmlParameterSerializingContext.cpp)
 
+GENERATE_EXPORT_HEADER( parameter
```

Comment:
For coding style consistency:

- change the command name to lower case
- remove the whitespace before the target name